src/lib/server/migrations.ts

```diff
@@ -26,7 +26,7 @@ const migrations = [
 		}
 	},
 	{
-		_id: new ObjectId('39811201e92e590e858af8ba'),
+		_id: new ObjectId('39811201e92e590e858af8bb'),
```

Comment:
Please don't reuse an existing migration, create an additional migration at the end